depmapomics/ccle.py

```diff
@@ -0,0 +1,1452 @@
+            dry_run=True,
+            gumbo=gumbo,
+        )
+    except:
+        print("no wes for this sampleset")
+
+    pr_table = track.update_pr_from_seq(trackerobj)
+    renaming_dict = dict(list(zip(pr_table.CDSID, pr_table.index)))
+    wgssegments_pr = (
+        wgssegments[wgssegments[SAMPLEID].isin(set(renaming_dict.keys()))]
+        .replace({SAMPLEID: renaming_dict})
+        .reset_index(drop=True)
+    )
+    wgscn_pr = genecn[genecn.index.isin(set(renaming_dict.keys()))].rename(
+        index=renaming_dict
+    )
+    wgssegments_pr.to_csv(folder + "segments_all_profile.csv", index=False)
+    wgscn_pr.to_csv(folder + "genecn_all_profile.csv")
+
+    print("merging PR-level seg file")
+    mergedsegments_pr = wgssegments_pr.append(wessegments_pr).reset_index(drop=True)
+    mergedsegments_pr = (
+        mergedsegments_pr[
+            [
+                SAMPLEID,
+                "Chromosome",
+                "Start",
+                "End",
+                "Segment_Mean",
+                "Num_Probes",
+                "Status",
+                "Source",
+            ]
+        ]
+        .sort_values(by=[SAMPLEID, "Chromosome", "Start", "End"])
+        .reset_index(drop=True)
+    )
+    mergedsegments_pr.loc[
+        mergedsegments_pr[mergedsegments_pr.Chromosome == "X"].index, "Status"
+    ] = "U"
+    # merging wes and wgs
+    folder = os.path.join("temp", samplesetname, "")
+    mergedsegments_pr = mut.manageGapsInSegments(mergedsegments_pr)
+    mergedsegments_pr.to_csv(folder + "merged_segments_all_profile.csv", index=False)
+
+    mergedgenecn_pr = mut.toGeneMatrix(mergedsegments_pr, mybiomart).apply(
+        lambda x: np.log2(1 + x)
+    )
+    mergedgenecn_pr.to_csv(folder + "merged_genecn_all_profile.csv")
+
+    # uploading to taiga
+    print("uploading to taiga")
+    tc.update_dataset(
+        changes_description="new "
+        + samplesetname
+        + " release! (removed misslabellings, see changelog)",
+        dataset_permaname=taiga_dataset,
+        upload_files=[
+            {
+                "path": folder + "wes_segments_all.csv",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "wes_genecn_all.csv",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "wgs_segments_all.csv",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "wgs_genecn_all.csv",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "wes_segments_all_profile.csv",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "wes_genecn_all_profile.csv",
```

Comment:
if we add pureCN it should be 4 more files.

Reply:
Yes but this would require ops to update the release dates for some wes lines, since some currently don't have them simply because they were ignored because there are wgs profiles under the same model. I created a task for this but I'll mark this as resolved here